SimPEG/EM/NSEM/RxNSEM.py

```diff
@@ -0,0 +1,696 @@
+""" Module RxNSEM.py
+
+Receivers for the NSEM problem
+
+"""
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import division
+
+from scipy.constants import mu_0
+
+import SimPEG
+from SimPEG import mkvc
+# from SimPEG.EM.Utils.EMUtils import mu_0
+
+
+class rx_Point_NSEM(SimPEG.Survey.BaseRx):
```

Comment:
Not strictly,  but kind of. I renamed it `BaseRxPoint` since if there are other types then `Point` the base will be different. We might want to give this more though with redesigned Rx's